### ISSUE TITLE
UX: Add a Help menu item to Main Menu !

### ISSUE THREAD

**Issue by aybe** [joncampbell123/dosbox-x#1844]
**Is your feature request related to a problem? Please describe.**

After #1842, it seems like many things can be solved by just reading the wiki.

**Describe the solution you'd like**

Great effort has been put in writing the wiki, why not make it easily accessible:

![dosbox-x_2020-09-11_00-19-11](https://user-images.githubusercontent.com/1537591/92815390-7b89f600-f3c4-11ea-9fc0-01a5ae9b5bfa.png)

**Comment by Wengier** [comment]
Great suggestion! Will definitely do!

**Comment by Wengier** [comment]
It is already done! The "Help" menu now includes items "Introduction", "DOSBox-X homepage", "DOSBox-X Wiki guide", "DOSBox-X support" and "About". Very useful I believe.

**Comment by aybe** [comment]
Amazing !!! 🥇

**Comment by aybe** [comment]
Just tried it, it's amazing, great work 💯 

Reference commit: a500b0372e57e0317bc5c3cd704f95c16f05e30b

**Comment by rderooy** [comment]
This seems to have broken the build on Linux.

```
epbase=`echo menu.o | sed 's|[^/]*$|.deps/&|;s|\.o$||'`;\
g++ -DHAVE_CONFIG_H -I. -I../..  -I../../include -I../../src/aviwriter "-DRESDIR=\"\"" -I../../src -I../../src/mt32 -Wno-int-to-void-pointer-cast   -Wno-address-of-packed-member   -Wno-format-zero-length   -Wno-missing-field-initializers   -Wno-strict-aliasing   -Wno-implicit-fallthrough   -Wno-deprecated-declarations   -Wconversion-null   -Wsign-promo   -Wlogical-op   -pedantic   -Wunused   -Wextra   -Wall   -I/usr/include/SDL2 -D_REENTRANT -I/home/robert/github/dosbox-x/vs2015/sdlnet/linux-host/include -I/home/robert/github/dosbox-x/vs2015/sdlnet/linux-host/include/SDL -I/usr/include/freetype2 -I/usr/include/libpng16   -g -std=gnu++11  -O2 -msse  -Wall   -Wextra   -Wunused   -pedantic   -Wlogical-op   -Wsign-promo   -Wconversion-null   -Wno-deprecated-declarations   -Wno-implicit-fallthrough   -Wno-strict-aliasing   -Wno-missing-field-initializers   -Wno-format-zero-length   -Wno-address-of-packed-member   -Wno-int-to-void-pointer-cast  -I/home/robert/github/dosbox-x/vs2015/sdlnet/linux-host/include -I/home/robert/github/dosbox-x/vs2015/sdlnet/linux-host/include/SDL -D_XOPEN_SOURCE=700 -D_POSIX_C_SOURCE=200809L  -MT menu.o -MD -MP -MF $depbase.Tpo -c -o menu.o menu.cpp &&\
mv -f $depbase.Tpo $depbase.Po
sdl_gui.cpp: In member function ‘virtual void ConfigurationWindow::actionExecuted(GUI::ActionEventSource*, const GUI::String&)’:
sdl_gui.cpp:1810:76: error: ‘SW_SHOWNORMAL’ was not declared in this scope
 1810 |             ShellExecute(NULL, "open", "http://dosbox-x.com/", NULL, NULL, SW_SHOWNORMAL);
      |                                                                            ^~~~~~~~~~~~~
sdl_gui.cpp:1810:13: error: ‘ShellExecute’ was not declared in this scope
 1810 |             ShellExecute(NULL, "open", "http://dosbox-x.com/", NULL, NULL, SW_SHOWNORMAL);
      |             ^~~~~~~~~~~~
At global scope:
cc1plus: note: unrecognized command-line option ‘-Wno-int-to-void-pointer-cast’ may have been intended to silence earlier diagnostics
cc1plus: note: unrecognized command-line option ‘-Wno-int-to-void-pointer-cast’ may have been intended to silence earlier diagnostics
make[3]: *** [Makefile:434: sdl_gui.o] Error 1
make[3]: *** Waiting for unfinished jobs....
sdlmain.cpp: In function ‘bool help_open_url_callback(DOSBoxMenu*, DOSBoxMenu::item*)’:
sdlmain.cpp:8270:79: warning: unused parameter ‘menuitem’ [-Wunused-parameter]
 8270 | bool help_open_url_callback(DOSBoxMenu * const menu, DOSBoxMenu::item * const menuitem) {
      |                                                      ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~
sdlmain.cpp: In function ‘bool help_intro_callback(DOSBoxMenu*, DOSBoxMenu::item*)’:
sdlmain.cpp:8284:45: warning: unused parameter ‘menu’ [-Wunused-parameter]
 8284 | bool help_intro_callback(DOSBoxMenu * const menu, DOSBoxMenu::item * const menuitem) {
      |                          ~~~~~~~~~~~~~~~~~~~^~~~
sdlmain.cpp:8284:76: warning: unused parameter ‘menuitem’ [-Wunused-parameter]
 8284 | bool help_intro_callback(DOSBoxMenu * const menu, DOSBoxMenu::item * const menuitem) {
      |                                                   ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~
sdlmain.cpp: In function ‘bool help_about_callback(DOSBoxMenu*, DOSBoxMenu::item*)’:
sdlmain.cpp:8298:45: warning: unused parameter ‘menu’ [-Wunused-parameter]
 8298 | bool help_about_callback(DOSBoxMenu * const menu, DOSBoxMenu::item * const menuitem) {
      |                          ~~~~~~~~~~~~~~~~~~~^~~~
sdlmain.cpp:8298:76: warning: unused parameter ‘menuitem’ [-Wunused-parameter]
 8298 | bool help_about_callback(DOSBoxMenu * const menu, DOSBoxMenu::item * const menuitem) {
      |                                                   ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~
At global scope:
cc1plus: note: unrecognized command-line option ‘-Wno-int-to-void-pointer-cast’ may have been intended to silence earlier diagnostics
cc1plus: note: unrecognized command-line option ‘-Wno-int-to-void-pointer-cast’ may have been intended to silence earlier diagnostics
make[3]: Leaving directory '/home/robert/github/dosbox-x/src/gui'
make[2]: *** [Makefile:527: all-recursive] Error 1
make[2]: Leaving directory '/home/robert/github/dosbox-x/src'
make[1]: *** [Makefile:395: all-recursive] Error 1
make[1]: Leaving directory '/home/robert/github/dosbox-x'
make: *** [Makefile:330: all] Error 2
```

**Comment by Wengier** [comment]
@rderooy Have you noticed the final commit which I added "#ifdef WIN32" and "fixme: Linux" notices last night? See  ded54acfff950e60d19423aa46f9ccb5a5c4f4e1. It was too late last night, and I was planning to fix it for Linux today. I believe your code is not the latest commit.

**Comment by rderooy** [comment]
No, I did not look at the commit :-) was doing other stuff.

In any case, I just did another pull, and git is reporting there is nothing new.
The previous errors, where from a SDL2 build, this is what I get with SDL1.

```
sdl_gui.cpp: In member function ‘virtual void ConfigurationWindow::actionExecuted(GUI::ActionEventSource*, const GUI::String&)’:
sdl_gui.cpp:1810:76: error: ‘SW_SHOWNORMAL’ was not declared in this scope
 1810 |             ShellExecute(NULL, "open", "http://dosbox-x.com/", NULL, NULL, SW_SHOWNORMAL);
      |                                                                            ^~~~~~~~~~~~~
sdl_gui.cpp:1810:13: error: ‘ShellExecute’ was not declared in this scope
 1810 |             ShellExecute(NULL, "open", "http://dosbox-x.com/", NULL, NULL, SW_SHOWNORMAL);
      |             ^~~~~~~~~~~~
At global scope:
cc1plus: note: unrecognized command-line option ‘-Wno-int-to-void-pointer-cast’ may have been intended to silence earlier diagnostics
cc1plus: note: unrecognized command-line option ‘-Wno-int-to-void-pointer-cast’ may have been intended to silence earlier diagnostics
make[3]: *** [Makefile:434: sdl_gui.o] Error 1
make[3]: Leaving directory '/home/robert/github/dosbox-x/src/gui'
make[2]: *** [Makefile:527: all-recursive] Error 1
make[2]: Leaving directory '/home/robert/github/dosbox-x/src'
make[1]: *** [Makefile:395: all-recursive] Error 1
make[1]: Leaving directory '/home/robert/github/dosbox-x'
make: *** [Makefile:330: all] Error 2
```

**Comment by Wengier** [comment]
@rderooy The actual fix for Linux is already on my computer, but I need to properly test it in Linux VM (which is slow) before pushing it.:-)

**Comment by Wengier** [comment]
@rderooy I have already pushed a commit for Linux etc. Seem to work in the Linux VM. Please check it on your system too.
